capstone-frontend/src/components/TestApi.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+
+export class TestApi extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      testData: null,
```

Comment:
When using React, it's generally best practice to initialize state fields with an empty version of the data type they'll hold. For example, if this will get populated with an object from an API response, initialize it as an empty object. For other types you can use an empty array, empty string, 0, etc.

Reply:
Ok.